src/test/java/MarlinDebug.java

```diff
@@ -124,9 +121,6 @@ public static void testCommit(final RepoInspection repoInspection, final String
                 Logger.info("  End processing {}", patch);
             }
         }
-
-        StringBuilder bigB = new StringBuilder();
-        LineGraphExport.toLineGraphFormat(commitDiff, bigB, DiffTreeMiner.MiningExportOptions(repoInspection.repo));
```

Comment:
This might have been here to test the filtering and transforming of DiffTrees but this class is deprecated anyway and we should probably delete it or at least flag it as `@Deprecated`.

Reply:
So how did it test that? Was there manual interaction involved? Or did it only check if the filtering or transformation crash?
This would be the second test class (besides `LinuxParsingTest`) annotated with `@Deprecated`.

Reply:
It was dome manually if I remember correctly. Dont take this code too serious. :D